app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt

```diff
@@ -27,4 +22,8 @@ class MainFragment : Fragment() {
         // TODO: Use the ViewModel
     }
 
+    companion object {
+        fun newInstance() = MainFragment()
+    }
+
```

Comment:
Moved companion object to the bottom of the class. https://kotlinlang.org/docs/reference/coding-conventions.html